kong/plugins/hmac-auth/access.lua

```diff
@@ -129,6 +149,22 @@ local function validate_clock_skew(headers, date_header_name, allowed_clock_skew
   return true
 end
 
+local function validate_request_body(headers, body)
```

Comment:
I can do one thing to abide by ieft draft.
At Kong, we can generate hash string prepended by the hash algorithm as recommended by draft, i.e.: 
`digest: SHA-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=\n`
Then we would need to mention it clearly that user needs to send digest header in this format and only `sha-256` algorithm is supported by Kong.